torch/nn/qat/modules/linear.py

```diff
@@ -39,11 +39,11 @@ def from_float(cls, mod, qconfig=None):
             Args: `mod` a float module, either produced by torch.quantization utilities
             or directly from user
         """
-        assert type(mod) == cls.__FLOAT_MODULE, ' qat.' + cls.__name__ + '.from_float only works for ' + \
-            cls.__FLOAT_MODULE.__name__
+        assert type(mod) == cls._FLOAT_MODULE, ' qat.' + cls.__name__ + '.from_float only works for ' + \
+            cls._FLOAT_MODULE.__name__
         if not qconfig:
             assert hasattr(mod, 'qconfig'), 'Input float module must have qconfig defined'
-            assert mod.qconfig, 'Input float module must has valid qconfig'
+            assert mod.qconfig, 'Input float module must have a valid qconfig'
```

Comment:
Do we need this check after  assert hasattr(mod, 'qconfig'). Is the second check ensuring that qconfig is not None?

Reply:
yeah this is to make sure we have a valid qconfig